.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "nextpnr-xilinx"]
 	path = nextpnr-xilinx
 	url = git@github.com:kintex-chatter/nextpnr-xilinx.git
+[submodule "prjxray"]
```

Comment:
Why add prjxray as a submodule? I consider it auxiliary set of binaries and data just like openFPGALoader or yosys that one would have to make available. I agree that setup of the toolchain is a pain these days, but see this to be in the scope of F4PGA (formerly known as Symbiflow), cf. https://github.com/kintex-chatter/xc7k325t-blinky-nextpnr/issues/18#issuecomment-1046351733.

Reply:
The main reason to add prjxray as a submodule is so that XRAY_DIR can be automatically set to a default value based on this know relative location for prjxray.